PersonBook/PersonBook/PersonBook/Program.cs

```diff
@@ -1,78 +1,88 @@
-﻿using PersonBook.Entity;
-using PersonBook.Service;
-using System;
-using System.Collections.Generic;
+﻿using System;
 
 namespace PersonBook
 {
+    using Entity;
+    using Service;
+
     class Program
     {
-       //
         static void Main(string[] args)
         {
-            PersonService pservice = new PersonService();
+            var personService = new PersonService();
+            bool flag = true;
 
-            while (true)
+            while (flag)
             {
-                Console.WriteLine("\n" + 
-               "~~~~~~Welcome to Person Book Application~~~~~\n " +
-               "Please Type\n " +
-               "1- List Persons\n " +
-               "2- Add new Person\n " +
-               "3- Update Person\n " +
-               "4- Delete Person\n " +
-               "5- exit\n");
+                PrintOptions();
                 string option = Console.ReadLine();
-              
+
                 switch (option)
```

Comment:
there shouldn't be any logic in switch case statement. 
```
switch(item){
  case 1:
    do();
   break;
case 2:
   doElse();
break;
}
```

Reply:
Done